test/extensions/access_loggers/wasm/config_test.cc

```diff
@@ -180,8 +180,11 @@ TEST_P(WasmAccessLogConfigTest, YamlLoadFromFileWasmInvalidConfig) {
   filter_instance->log(nullptr, nullptr, nullptr, log_stream_info);
 
   TestUtility::loadFromYaml(invalid_yaml, proto_config);
-  filter_instance = factory->createAccessLogInstance(proto_config, nullptr, context_);
-  filter_instance->log(nullptr, nullptr, nullptr, log_stream_info);
+  // When trying again with invalid config, it should raise the exception again because the canary
+  // will fail.
+  EXPECT_THROW_WITH_MESSAGE(factory->createAccessLogInstance(proto_config, nullptr, context_),
+                            Envoy::Extensions::Common::Wasm::WasmException,
+                            "Unable to create Wasm access log ");
```

Comment:
Note that we already verify that this exception is thrown earlier in this test (there are: `invalid_yaml` and `valid_yaml` cases), so perhaps this should be completely removed instead.

(Same in the other 2 tests)

Reply:
OK. it makes sense.
Will change it.